<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>simple_uml_editorClass</class>
 <widget class="QMainWindow" name="simple_uml_editorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>884</width>
    <height>674</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>simple_uml_editor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="select_Button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>mouse.png</normaloff>mouse.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="ass_line_Button">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>line.png</normaloff>line.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>35</width>
      <height>35</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="gen_line_Button">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>generalization_line.png</normaloff>generalization_line.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>35</width>
      <height>35</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="com_line_Button">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>composition_line.png</normaloff>composition_line.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>35</width>
      <height>35</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="class_Button">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>class.png</normaloff>class.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>35</width>
      <height>35</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="usecase_Button">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>use_case.png</normaloff>use_case.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>35</width>
      <height>35</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>811</width>
      <height>501</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>884</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="separator"/>
    <addaction name="Change_object_name"/>
    <addaction name="Group"/>
    <addaction name="Ungroup"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionFile">
   <property name="text">
    <string>file</string>
   </property>
  </action>
  <action name="Change_object_name">
   <property name="text">
    <string>change object name</string>
   </property>
  </action>
  <action name="Group">
   <property name="text">
    <string>group</string>
   </property>
  </action>
  <action name="Ungroup">
   <property name="text">
    <string>ungroup</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="simple_uml_editor.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>select_Button</sender>
   <signal>clicked()</signal>
   <receiver>simple_uml_editorClass</receiver>
   <slot>select_button_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>33</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>12</x>
     <y>112</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ass_line_Button</sender>
   <signal>clicked()</signal>
   <receiver>simple_uml_editorClass</receiver>
   <slot>ass_line_button_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>100</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel">
     <x>93</x>
     <y>96</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gen_line_Button</sender>
   <signal>clicked()</signal>
   <receiver>simple_uml_editorClass</receiver>
   <slot>gen_line_button_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>159</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>165</x>
     <y>100</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>com_line_Button</sender>
   <signal>clicked()</signal>
   <receiver>simple_uml_editorClass</receiver>
   <slot>com_line_button_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>243</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>class_Button</sender>
   <signal>clicked()</signal>
   <receiver>simple_uml_editorClass</receiver>
   <slot>class_button_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>288</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>100</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>usecase_Button</sender>
   <signal>clicked()</signal>
   <receiver>simple_uml_editorClass</receiver>
   <slot>usecase_button_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>345</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>394</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>select_button_clicked()</slot>
  <slot>ass_line_button_clicked()</slot>
  <slot>gen_line_button_clicked()</slot>
  <slot>com_line_button_clicked()</slot>
  <slot>class_button_clicked()</slot>
  <slot>usecase_button_clicked()</slot>
 </slots>
</ui>
